SU2_CFD/src/iteration/CIteration.cpp

```diff
@@ -0,0 +1,260 @@
+/*!
+ * \file iteration_structure.cpp
+ * \brief Main subroutines used by SU2_CFD
+ * \author F. Palacios, T. Economon
+ * \version 7.0.5 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2020, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "../include/iteration/CIteration.hpp"
+
+#include "../include/output/COutput.hpp"
+#include "../include/solvers/CFEASolver.hpp"
+
+CIteration::CIteration(const CConfig* config) {
+  rank = SU2_MPI::GetRank();
+  size = SU2_MPI::GetSize();
+
+  nInst = config->GetnTimeInstances();
+  nZone = config->GetnZone();
+
+  multizone = config->GetMultizone_Problem();
+  singlezone = !(config->GetMultizone_Problem());
+}
+
+CIteration::~CIteration(void) {}
+
```

Comment:
Default the destructor in the hpp "the best kind of documentation is the one the compiler understands".